bin/linaria.js

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const mkdir = require('mkdirp');
+const parseArgs = require('minimist');
+const processFile = require('../lib/node');
+
+const args = parseArgs(process.argv.slice(2));
+
+if (args.h || args.help) {
+  console.log(
+    [
+      'Usage: linaria [options] <file1> [...<fileN>]',
+      '',
+      'Arguments',
+      '  <file1> [...<fileN>]  File paths or glob patterns',
+      '',
+      'Options:',
+      '  --help, -h            Show help                        [boolean]',
+      '  --source-maps, -s     Generate source maps             [boolean]',
+      '  --require-css, -r     Require CSS in original JS file  [boolean]',
+      '  --out-dir, -o         Output directory                 [string]',
+      '  --config, -c          Path to Babel config file         string]',
```

missing brackets?

---

also should mention which options are required and throw if not provided. I'd recommend using something like yargs to make it easier instead of writing it manually https://github.com/yargs/yargs

---

@zamotany `--out-dir` should be required, no? otherwise would be too easy to pollute working directory with css files by mistake.

---

It would be, I think babel for example doesn't require you to provide out dir. Or maybe we should set some default like `build`? or `css_dist`?

---

maybe `extracted_css` or something. but maybe better just to require it. I'd want to change it anyways.